src/applications/edu-benefits/complaint-tool/config/form.js

```diff
@@ -283,7 +283,8 @@ const formConfig = {
                 },
                 'ui:errorMessages': {
                   atLeastOne: 'Please select at least one'
-                }
+                },
+                'ui:required': () => true
```

Comment:
Since this is not conditional we can set `required:  ['programs']` on the educationDetails schema for this. 
This should have been carried over when this schema was nested but was missed https://github.com/department-of-veterans-affairs/vets-website/pull/8174/files#diff-e03d34052d066c0db8877c8048b7a1f3L304